src/Event/LocationMarkedAsDuplicateProcessManager.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event;
+
+use Broadway\CommandHandling\CommandBusInterface;
+use Broadway\Domain\DomainMessage;
+use Broadway\EventHandling\EventListenerInterface;
+use CultuurNet\UDB3\Event\Commands\UpdateLocation;
+use CultuurNet\UDB3\Event\ReadModel\Relations\RepositoryInterface;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Place\Events\MarkedAsDuplicate;
+
+final class LocationMarkedAsDuplicateProcessManager implements EventListenerInterface
+{
+    /**
+     * @var RepositoryInterface
+     */
+    private $relationsRepository;
+
+    /**
+     * @var CommandBusInterface
+     */
+    private $commandBus;
+
+    public function __construct(
+        RepositoryInterface $relationsRepository,
+        CommandBusInterface $commandBus
+    ) {
+        $this->relationsRepository = $relationsRepository;
+        $this->commandBus = $commandBus;
+    }
+
+    public function handle(DomainMessage $domainMessage)
+    {
+        $domainEvent = $domainMessage->getPayload();
+
+        // Only handle (Place)MarkedAsDuplicate events.
+        if (!($domainEvent instanceof MarkedAsDuplicate)) {
+            return;
+        }
+
+        $duplicatePlaceId = $domainEvent->getPlaceId();
+        $canonicalPlaceId = $domainEvent->getDuplicateOf();
```

Comment:
Should we call these getters `getDuplicatePlaceId()` and `getCanonicalPlaceId()` for clarity?

Reply:
I'm fine with either. Do you have a preference?

Reply:
I would prefer the duplicate/canonical if possible

Reply:
I'll refactor this in a follow-up once both your PRs are merged

Reply:
Never mind, the getPlaceId is part of the `PlaceEvent` abstraction